go.mod

```diff
@@ -64,8 +64,8 @@ require (
 	github.com/nfnt/resize v0.0.0-20180221191011-83c6a9932646
 	github.com/pion/mediadevices v0.4.0
 	github.com/pion/rtp v1.7.13
-	github.com/pixiv/go-libjpeg v0.0.0-20190822045933-3da21a74767d
 	github.com/pion/webrtc/v3 v3.1.54
+	github.com/pixiv/go-libjpeg v0.0.0-20190822045933-3da21a74767d
```

Comment:
we don't want to use this library in go.mod, potentially you'll want to do a rebase so you don't accidently add it.